docs/processors/standalone/how-it-works.mdx

```diff
@@ -198,11 +220,20 @@ running.
 - `4,294,967,293` - unknown command request. Internal error in Conduit, where
   the command request failed to be marshaled into a proto message.
 - `4,294,967,292` - unknown command response. Internal error in Conduit, where
-  the command response failed to be unmarshaled from the proto message into a
+  the command response failed to be unmarshalled from the proto message into a
   struct.
 - `4,294,967,291` - memory out of range. Internal error in Conduit, where it
   tried to write into the allocated memory, but failed to do so, as the size was
   insufficient.
+- `4,294,967,290` - Internal error in Conduit, could be an error while 
+  marshalling/unmarshalling the buffer, or other reasons, the specific
+  error should be logged by conduit.
```

Comment:
```suggestion
  error should be logged by Conduit.
```

Reply:
answering your question, since you both already reviewed this one, I'll add the examples in another PR, they are for another page on the docs so not really related to this one.